src/handlers/wallet/handler.rs

```diff
@@ -226,6 +238,40 @@ async fn handle_rpc(
             .map_err(Error::GetAssets)?,
         )
         .map_err(|e| Error::Internal(InternalError::SerializeResponse(e))),
+        PAY_GET_EXCHANGES => serde_json::to_value(
+            &get_exchanges::handler(
+                state,
+                connect_info,
+                headers,
+                Query(get_exchanges::QueryParams {
+                    sdk_info: query.sdk_info,
+                }),
+                Json(
+                    serde_json::from_value::<get_exchanges::GetExchangesRequest>(params)
```

Comment:
The type here should be inferable

```suggestion
                    serde_json::from_value(params)
```